helpers.go

```diff
@@ -19,7 +20,8 @@ import (
 func TestString(t *testing.T, name string) string {
 	t.Helper()
 
-	n := rand.Int() //nolint:gosec
+	r := rand.NewSource(time.Now().UnixNano())
```

Comment:
why?

Reply:
Because, somewhat surprisingly, it generates better randomness this way. Reseeding should be done rarely if ever, compared to using a generator.

But more importantly, the global source is available in any case, whatever we use it there or not. Some other part may use by accident, and it will not be seeded then. It is somewhat similar to how we use package-level Swagger clients that are always there instead of creating a new instance.